packages/api/src/index.js

```diff
@@ -26,6 +26,7 @@ router.post('/user/tokens', withCorsHeaders(withAuth(userTokensPost)))
 router.delete('/user/tokens/:id', withCorsHeaders(withAuth(userTokensDelete)))
 router.get('/user/uploads', withCorsHeaders(withAuth(userUploadsGet)))
 router.delete('/user/uploads/:cid', withCorsHeaders(withAuth(userUploadsDelete)))
+router.get('/user/storage', withCorsHeaders(withAuth(userStorageGet)))
```

Comment:
I'm tempted to just make this into a more generic user account info endpoint (that only returns storage usage for now). So like `/user/account`. We may have more user account things to retrieve in the future (IDK maybe we want to retrieve a user's avatar or something). WDYT?

Reply:
`/user/account` seems a good plan